CITATION.cff

```diff
@@ -0,0 +1,124 @@
+cff-version: 1.2.0
+message: "If you use this book, please cite it as below."
+title: "SnowEx Hackweek JupyterBook Tutorials"
+version: 2021.07.23
+date-released: "2021-07-23"
+identifiers:
+- description: "collection of archived snapshots of all versions"
+  type: doi
+  value: TODO
+  license: MIT
+  repository-code: https://github.com/snowex-hackweek/website
+authors:
+- family-names: Henderson
+  given-names: Scott
+  orcid: "https://orcid.org/0000-0003-0624-4965"
```

Comment:
I did my best to find everyone on the 'team' page https://snowex-hackweek.github.io/website/team.html by name and affiliation on https://orcid.org, but some are missing.